distribution/docs/src/main/resources/content/_managing/_securing/managing-keystores.adoc

```diff
@@ -9,9 +9,16 @@
 
 Certificates, and sometimes their associated private keys, are stored in keystore files.
 ${branding} includes two default keystore files, the server key store and the server trust store.
-The server keystore holds the certificates and private keys that ${branding} uses to identify itself to other nodes on the network.
+The server keystore holds ${branding}'s certificate and private key.
+It will also hold the certificates of other nodes whose signature ${branding} will accept.
 The truststore holds the certificates of nodes or other entities that ${branding} needs to trust.
 
+[NOTE]
+====
+Individual certificates of other nodes should be added to the keystore instead of CA certificates.
+If a CA's certificate is added, ${branding} will automatically trust any certificate that is signed by that CA.
+====
```

Comment:
✏️ This message could be of higher urgency, for instance a warning, since adding a CA to the keystore makes the system accessible by any cert signed with that CA.